Source/ASNetworkImageNode.mm

```diff
@@ -451,18 +465,26 @@ - (void)_cancelDownloadAndClearImage
   [self _cancelImageDownload];
```

Comment:
Yeah, I'm actually concerned about changing this behavior. I'm not sure we're consistent, but I think it makes since to be consistent and always `_cancelDownloadAndClearImage` as part of a locked transaction. We should probably add the `_locked_` prefix?